akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMultiMap.scala

```diff
@@ -185,16 +204,19 @@ final class ORMultiMap[A, B] private[akka] (
   }
 
   /**
-   * Scala API: Add an element to a set associated with a key. If there is no existing set then one will be initialised.
+   * Add an element to a set associated with a key. If there is no existing set then one will be initialised.
+   * TODO add implicit after deprecated is EOL.
```

Comment:
ok, yes not important

Reply:
I think it is important but as there was no longer a Java vs Scala api, as I recall - I’m in an ✈️  on my phone - it didn’t apply. May change if we EOL he deprecated function eventually.